nifi-nar-bundles/nifi-social-media-bundle/nifi-twitter-processors/src/main/java/org/apache/nifi/processors/twitter/ConsumeTwitter.java

```diff
@@ -0,0 +1,366 @@
+            "Streams about one percent of all Tweets. " +
+                    "https://developer.twitter.com/en/docs/twitter-api/tweets/volume-streams/api-reference/get-tweets-sample-stream");
+    static final AllowableValue ENDPOINT_SEARCH = new AllowableValue(StreamEndpoint.SEARCH_ENDPOINT.getEndpointName(),
+            "Search Stream",
+            "The endpoint that provides a stream of tweets that matches the rules you added to the stream. " +
+                    "If rules are not configured, then the stream will be empty. " +
+                    "https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/api-reference/get-tweets-search-stream");
+
+    public static final PropertyDescriptor ENDPOINT = new PropertyDescriptor.Builder()
+            .name("stream-endpoint")
+            .displayName("Stream Endpoint")
+            .description("Specifies which endpoint tweets should be streamed. " +
+                    "Usage of search endpoint requires that rules be uploaded beforehand. See " +
+                    "https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/api-reference/" +
+                    "post-tweets-search-stream-rules")
+            .required(true)
+            .allowableValues(ENDPOINT_SAMPLE, ENDPOINT_SEARCH)
+            .defaultValue(ENDPOINT_SAMPLE.getValue())
+            .build();
+    public static final PropertyDescriptor BASE_PATH = new PropertyDescriptor.Builder()
+            .name("base-path")
+            .displayName("Base Path")
+            .description("Specifies which base path the API client will use for HTTP requests. " +
+                    "Generally should not be changed from the default https://api.twitter.com except for testing")
+            .required(true)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .defaultValue("https://api.twitter.com")
+            .build();
+    public static final PropertyDescriptor BEARER_TOKEN = new PropertyDescriptor.Builder()
+            .name("bearer-token")
+            .displayName("Bearer Token")
+            .description("The Bearer Token provided by Twitter.")
+            .required(true)
+            .sensitive(true)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+    public static final PropertyDescriptor QUEUE_SIZE = new PropertyDescriptor.Builder()
+            .name("queue-size")
+            .displayName("Queue Size")
+            .description("Maximum size of internal queue for streamed messages")
+            .required(true)
+            .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
+            .defaultValue("10000")
+            .build();
+    public static final PropertyDescriptor BATCH_SIZE = new PropertyDescriptor.Builder()
+            .name("batch-size")
+            .displayName("Batch Size")
+            .description("The maximum size of the number of tweets to be written to a single FlowFile." +
```

Comment:
Additional space needed:
```suggestion
            .description("The maximum size of the number of Tweets to be written to a single FlowFile. " +
```

Reply:
Making the changes